src/test/prelude.rs

```diff
@@ -6,13 +6,18 @@ pub use serde_json::json;
 use std::future::Future;
 pub use web3::api::Web3;
 
-/// An extension future to assert that a future resolves immediately.
-pub trait FutureImmediateExt: Future {
+/// An extension future to wait for a future.
+pub trait FutureTestExt: Future {
+    /// Block thread on a future completing.
+    fn wait(self) -> Self::Output;
     /// Block thread on a future completing.
```

Comment:
Just realized this doc comment is wrong from before, so you mind changing it here already?